File: tasks/michal-zawadzki/dodawanie.py
tablica = [1, 2, 3, 4, 5]


class Liczba:
    def __init__(self, wartosc):
        self.liczba = wartosc

    def Podaj(self):
        return self.liczba


def Suma(a, b):
    return tablica[a.Podaj()] + tablica[b.Podaj()]

File: tasks/Maciej-Dominiak/dodawanie.js
//JS


class Podawak
{
	constructor(wartosc)
	{
		this.wartosc = wartosc;
	}
	
	Podaj()
	{
		return this.wartosc;
	}
}

const tablica = [0,1,2,3];

function Suma(a,b)
{
	return tablica[a.Podaj()]+tablica[b.Podaj()];
}